### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by SamYuan1990**
try to add i18n support for document.

### CODE REVIEW

File: .github/workflows/mkdocs-ghpages.yaml [hunk 1]
```diff
@@ -10,5 +10,5 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.x
-      - run: pip install mkdocs-material 
+      - run: pip install mkdocs-material mkdocs-static-i18n
```
Comment (rootfs) [inline .github/workflows/mkdocs-ghpages.yaml:13]: how about babel?
https://www.mkdocs.org/dev-guide/translations/

Comment (SamYuan1990) [reply .github/workflows/mkdocs-ghpages.yaml:13]: https://github.com/mkdocs/mkdocs/wiki/MkDocs-Plugins#i18n
https://github.com/mkdocs/catalog
https://github.com/ultrabug/mkdocs-static-i18n
and the user guide can be found there.

I suppose we just need follow user guide right?
https://www.mkdocs.org/user-guide/localizing-your-theme/
hence... do we need babel?

File: docs/index.zh.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Kubernetes Efficient Power Level Exporter (Kepler)
+
+Kepler (Kubernetes-based Efficient Power Level Exporter)是一个prometheus exportor。
+它通过eBPF技术与CPU性能计数器以及Linux内核的tracepoints进行交互。
+
+这些数据和来自cgroup或者sysfs的信息一起可以放入机器学习模型中来对pod的能耗进行预测。
```
Comment (jichenjc) [inline docs/index.zh.md:6]: pod and process ?

File: docs/index.zh.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Kubernetes Efficient Power Level Exporter (Kepler)
+
+Kepler (Kubernetes-based Efficient Power Level Exporter)是一个prometheus exportor。
+它通过eBPF技术与CPU性能计数器以及Linux内核的tracepoints进行交互。
```
Comment (jichenjc) [inline docs/index.zh.md:4]: 交互以获得特定的数据？

**Comment by SamYuan1990** [comment]
@rootfs , @jichenjc please help review this PR and let's try to enable zh document for kepler. :-)

**Review by rootfs — APPROVED** [review]

**Pull request merged**